### ISSUE TITLE
ns-render asserts fail for splatfactow.splatfactow_datamanager.SplatfactoWDatamanagerConfig

### ISSUE THREAD

**Issue by Parskatt** [KevinXu02/splatfacto-w#18]
See also https://github.com/nerfstudio-project/nerfstudio/issues/2913

Perhaps you could subclass SplatfactoWDatamanagerConfig from either VanillaDataManagerConfig or FullImageDatamanagerConfig?

The asserts that fail are https://github.com/nerfstudio-project/nerfstudio/blob/fc4fc5cb15ad994ea82d8c651c9d42172d890de1/nerfstudio/scripts/render.py#L739

Simply commenting these assertions are works for me, and the renders look fine, so it should be fine to subclass, or would something weird be inherited?

By the way, nice work.

**Comment by StevePotter** [comment]
Same thing happening to me:

I run `ns-train splatfacto-w-light ...`

And get this:
```
AssertionError: pipeline.datamanager.dataparser was provided a default value of type <class 'splatfactow.nerfw_dataparser.NerfWDataParserConfig'> but no matching subcommand was found. A type may be missing in the Union type declaration for pipeline.datamanager.dataparser, which currently expects [typing.Annotated[nerfstudio.data.dataparsers.nerfstudio_dataparser.NerfstudioDataParserConfig, _SubcommandConfiguration(name='nerfstudio-data', default=NerfstudioDataParserConfig(_target=<class 
....
```

When I remove `splatfactow = 'splatfactow.splatfactow_config:splatfactow_config'` from pyproject.toml, it works

**Comment by StevePotter** [comment]
I fixed the problem, at least for me, in #20

**Comment by KevinXu02** [comment]
Fixed in https://github.com/KevinXu02/splatfacto-w/pull/20
